pwndbg/gdblib/disasm/x86.py

```diff
@@ -330,12 +330,18 @@ def _parse_memory(self, instruction: PwndbgInstruction, op: EnhancedOperand, emu
         # Get memory address (Ex: lea    rax, [rip + 0xd55], this would return $rip+0xd55. Does not dereference)
         target = 0
 
-        # There doesn't appear to be a good way to read from segmented
-        # addresses within GDB.
         if op.mem.segment != 0:
```

Comment:
Given that we are doing a lot of accesses into `op.mem` shall we maybe assign `op.mem` to `m` or sth?